macosx.gypi

```diff
@@ -22,11 +22,5 @@
             'NN_USE_EVENTFD',
             'NN_HAVE_MSG_CONTROL'
         ],
-        'include_dirs': [
-          'deps/nanomsg/src',
-        ],
-        'cflags': [
-          '-O3 -D_THREAD_SAFE'
```

Comment:
looks like it was removed even: https://github.com/nanomsg/nanomsg/search?utf8=%E2%9C%93&q=_THREAD_SAFE